.github/workflows/gobo_format.yml

```
@@ -0,0 +1,69 @@
+
+      - name: Extract ZIP
+        run: |
+          unzip gobo-ubuntu.zip
+
+      - name: Ensure executable permissions
+        run: |
+          chmod +x ./gobo
+
+      - name: Get Changed Files
+        id: changed_files
+        run: |
```

actionlint (1.7.4)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)